docs/writing-tests/interaction-testing.md

```diff
@@ -6,17 +6,52 @@ As you build more complex UIs like pages, components become responsible for more
 
 In a nutshell, you start by supplying the appropriate props for the initial state of a component. Then simulate user behavior such as clicks and form entries. Finally, check whether the UI and component state update correctly.
 
+In Storybook, this familiar workflow happens in your browser. That makes it easier to debug failures because you're running tests in the same environment as you develop components—the browser.
+
 ![Storybook interaction testing](./storybook-interaction-tests.gif)
 
-## Setup interactions addon
+## How does component testing in Storybook work?
 
-You can set up interaction testing in Storybook using the `play` function and [`@storybook/addon-interactions`](https://storybook.js.org/addons/@storybook/addon-interactions/).
+You start by writing a [**story**](../writing-stories/introduction.md) to set up the component's initial state. Then simulate user behavior using the **play** function. Finally, use the Storybook **test-runner** to verify the DOM structure. Additionally, you can automate test execution via the [command line](./test-runner.md#cli-options) or in your [CI environment](./test-runner.md#set-up-ci-to-run-tests).
```

Comment:
> test-runner

Other pages in this PR use "test runner". I prefer the hyphenated version, but the most important thing is to be consistent.

Reply:
> to verify the DOM structure

I'm not sure if that's the correct way of phrasing this. The out-of-the-box experience with the test runner does not test the DOM structure. It tests whether the component has rendered correctly, and additionally it tests if the interaction tests are passing in case you have the play function with tests

Reply:
addressed with the latest commit.